### PR TITLE
feat(android): add "Don't keep activities" functionality

### PR DESCRIPTION
**Pull request by vchimev**
This allows you to turn on/off "Don't keep activities" setting in the Developer options for Android.

### CODE REVIEW

File: lib/android-controller.ts [hunk 1]
```diff
@@ -583,6 +583,23 @@ export class AndroidController {
...
     }
+
+    private static getAlwaysFinishActivitiesGlobalSettingsValue(device: IDevice): boolean {
+        const prefix = AndroidController.gettokenPrefix(device.type);
+        return executeCommand(AndroidController.ADB + " -s " + `${prefix}${device.token}` + " shell settings get global always_finish_activities").trim() === "1";
+    }
+
+    public static setDontKeepActivities(value: boolean, device: IDevice) {
+        let status: 0 | 1;
+        value ? status = 1 : status = 0;
```
Comment (sis0k0) [inline lib/android-controller.ts]: const status = value ? 1 : 0;

File: lib/android-controller.ts [hunk 2]
```diff
@@ -583,6 +583,23 @@ export class AndroidController {
...
+
+        const prefix = AndroidController.gettokenPrefix(device.type);
+        executeCommand(AndroidController.ADB + ` -s ${prefix}${device.token} shell service call activity 43 i32 ${status}`);
```
Comment (sis0k0) [inline lib/android-controller.ts]: \`${AndroidController.ADB} -s ${prefix}${device.token} shell service call activity 43 i32 ${status}\`

File: lib/android-controller.ts [hunk 3]
```diff
@@ -583,6 +583,23 @@ export class AndroidController {
...
+
+        if (this.getAlwaysFinishActivitiesGlobalSettingsValue(device) !== value) {
+            throw new Error(`Failed to set \"Don't keep activities\" to ${value}!`);
```
Comment (sis0k0) [inline lib/android-controller.ts]: You don't need to escape the quotes when using backticks (`).

File: lib/android-controller.ts [hunk 4]
```diff
@@ -583,6 +583,23 @@ export class AndroidController {
     private static gettokenPrefix(type: DeviceType) {
         return type === DeviceType.DEVICE ? "" : "emulator-";
     }
+
+    private static getAlwaysFinishActivitiesGlobalSettingsValue(device: IDevice): boolean {
+        const prefix = AndroidController.gettokenPrefix(device.type);
+        return executeCommand(AndroidController.ADB + " -s " + `${prefix}${device.token}` + " shell settings get global always_finish_activities").trim() === "1";
```
Comment (sis0k0) [inline lib/android-controller.ts]: You can move the command to a variable.
Also you can move the result of the execution to a variable.
```
AndroidController.ADB + " -s " + `${prefix}${device.token}` + " shell settings get global always_finish_activities"
```
-->
```
`${AndroidController.ADB} -s ${prefix}${device.token} shell settings get global always_finish_activities`
```

**Comment by vchimev** [comment]
Ping @SvetoslavTsenov and @sis0k0.

**Pull request merged**